packages/jupyter-ai-magics/jupyter_ai_magics/providers.py

```diff
@@ -17,6 +17,8 @@
     Union,
 )
 
+import anthropic
+import openai
```

Comment:
Having these as top imports means, we can not ship Jupyter AI without including these as dependencies. We should rather move these imports to the respective provider functions.